blueox/__init__.py

```diff
@@ -30,20 +30,26 @@
 from .errors import Error
 from .logger import LogHandler
 from .timer import timeit
+from .recorders import kafka_recorder
 
 log = logging.getLogger(__name__)
 
+OVERRIDE_KAFKA_RECORDER = os.getenv('BLUEOX_OVERRIDE_KAFKA_RECORDER', 0)
+
 
 def configure(host, port, recorder=None):
     """Initialize blueox
 
-    This instructs the blueox system where to send it's logging data. If blueox is not configured, log data will
+    This instructs the blueox system where to send its logging data. If blueox is not configured, log data will
     be silently dropped.
 
     Currently we support logging through the network (and the configured host and port) to a blueoxd instances, or
     to the specified recorder function
     """
-    if recorder:
+    if int(OVERRIDE_KAFKA_RECORDER) == 1:
```

Comment:
not gonna lie, not a fan of magical overrides via env vars. Why not just have it come in as a recorder?

Reply:
Yeah, fair. I was hoping to avoid a bunch of edits throughout postal changing how it's called, but it would be simple enough to call
```py
import blueox
from blueox.recorders import kafka

blueox.configure(recorder=kafka.send)
```

Reply:
oh huh, wait, if thats the case no go back to your original way lol